SPEC.md

```diff
@@ -72,6 +72,25 @@ properties:
       Describes the current state of an operation.
 ```
 
+### Links
```

Comment:
I wouldn't put this in the schema section. These objects are never serialized in JSON form.

I would add another section to the doc called `Links` after [Predefined Handler Errors](https://github.com/nexus-rpc/api/blob/4d0ed524d8277e65823d2565974705958330d29e/SPEC.md#predefined-handler-errors)

Reply:
In the new section you can borrow the text from MDN:

The Nexus-Link header field provides a means for serializing one or more links in HTTP headers. This header is encoded the same way as the HTTP header `Link` described
[here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link).
Handlers and callers can specify links in different Nexus requests to associate an operation with arbitrary resources.